tensorflow/python/keras/engine/data_adapter_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorflow.python.keras.engine.data_adapter."""
+
+import tensorflow as tf
```

Comment:
Oh, we cannot have this hourglass import. It basically needs to load the entirety of TensorFlow in order to run the tests.

You should be able to only import the proper internal paths instead (in both test files)